### ISSUE TITLE
Gracefully detect other Fusion code styles

### ISSUE THREAD

**Issue by neos-bot** [neos/neos-development-collection#1104]
Jira issue originally created by user @skurfuerst:

```
prototype(Vendor.Site:Foo)
{
   bar = ""
}
```

must lead at least to a good error message. Ideally, this is understood by the parser as well.

 Jira-URL: https://jira.neos.io/browse/NEOS-1103

**Comment by rolandschuetz** [comment]
Fusion fails with a syntax error, that seems sufficient to me.

`Syntax error in line 8.
({)`

@skurfuerst Close?

**Comment by mhsdesign** [comment]
@mficzel and @jonnitto what do you say as fusionists?
I thought a lot about this in the parser rewrite, but it does need some ugly lookahead - i did archive it but ended up discarding it for now since its not part of the current spec and made some things more readable.

**Comment by mficzel** [comment]
Ahhh ... it fails because fusion wants to have the opening curly braces on the same line 

... i consider this ok as this is often part of the fusion syntax. 

```
foo  = Some:Prototye {
   ... 
} 
```

So unless we find a definition that allows to put opening curly braces always on the next line i would just leave it as it is and close this.

**Comment by mhsdesign** [comment]
Finally only after 7 years with https://github.com/neos/neos-development-collection/pull/3948 :P
